docs/book/src/reference/labels_and_annotations.md

```diff
@@ -1,62 +1,62 @@
+| cluster.x-k8s.io/cluster-name                                    | It is set on nodes identifying the name of the cluster the node belongs to.  | Cluster API | Nodes |
+| cluster.x-k8s.io/cluster-namespace                               | It is set on nodes identifying the namespace of the cluster the node belongs to.       | Cluster API | Nodes |
+| cluster.x-k8s.io/delete-machine                                  | It marks control plane and worker nodes that will be given priority for deletion when KCP or a MachineSet scales down. It is given top priority on all delete policies.   | User | Machines |
+| cluster.x-k8s.io/disable-machine-create                          | It can be used to signal a MachineSet to stop creating new machines. It is utilized in the OnDelete MachineDeploymentStrategy to allow the MachineDeployment controller to scale down older MachineSets when Machines are deleted and add the new replicas to the latest MachineSet.   |  Cluster API | Machineset |
+| cluster.x-k8s.io/managed-by                                      | It can be applied to InfraCluster resources to signify that some external system is managing the cluster infrastructure. Provider InfraCluster controllers will ignore resources with this annotation. An external controller must fulfill the contract of the InfraCluster resource. External infrastructure providers should ensure that the annotation, once set, cannot be removed.     | User | InfraCluster resources |
+| cluster.x-k8s.io/machine                                         | It is set on nodes identifying the machine the node belongs to. | Cluster API | Nodes |
+| cluster.x-k8s.io/owner-kind                                      | It is set on nodes identifying the machine's owner kind the node belongs to.     | Cluster API | Nodes |
+| cluster.x-k8s.io/owner-name                                      | It is set on nodes identifying the machine's owner name the node belongs to.    | Cluster API | Nodes |
+| cluster.x-k8s.io/paused                                          | It can be applied to any Cluster API object to prevent a controller from processing a resource. Controllers working with Cluster API objects must check the existence of this annotation on the reconciled object.  | User | Any Cluster API resource |
+| cluster.x-k8s.io/remediate-machine                               | It can be applied to a machine to manually mark it for remediation by MachineHealthCheck reconciler.   |    User          |    Machine         |
+|| cluster.x-k8s.io/replicas-managed-by                             | It can be applied to MachinePool resources to signify that some external system is managing infrastructure scaling for that pool. See [the MachinePool documentation](../developer/architecture/controllers/machine-pool.md#externally-managed-autoscaler) for more details.  | Infrastructure Providers | MachinePool resources |
```

Comment:
```suggestion
| cluster.x-k8s.io/replicas-managed-by                             | It can be applied to MachinePool resources to signify that some external system is managing infrastructure scaling for that pool. See [the MachinePool documentation](../developer/architecture/controllers/machine-pool.md#externally-managed-autoscaler) for more details.  | Infrastructure Providers | MachinePool resources |
```
(the formatting seems of)